### PR TITLE
Make setup and migrate work

### PR DESCRIPTION
**Pull request by tahini**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
         "packages/transition-frontend"
     ],
     "scripts": {
-        "create-project": "yarn workspace transition-legacy run create-project",
```
Comment (greenscientist) [inline package.json:13]: Is there code to remove as well or this was broken already?

**Comment by tahini** [comment]
Tested with a new database and it works

**Review by greenscientist — COMMENTED** [review]
You can see my comment in #442, I don't think we should have all the legacy migration script in the new repo. 
Maybe we add them and remove them after, but it's old stuff that we don't need to maybe, as the app don't exist publicly.

**Comment by tahini** [comment]
I agree it should not all be here. But we need to rethink our architecture for the multiple app support (evolution is quite independent, but osc2ar and octavi use some of the transition concepts). So either multiple migration tables or multiple databases/schemas.

To ease the transition from the legacy repo though, the easiest way is this PR, then have transition-legacy support the external chaire-lib/transition, _then_ fix #442, in which case, the current migrations will be removed and replaced with something else

**Comment by greenscientist** [comment]
We don't have to care about octavi and oscar. They don't exist in the world here. They are legacy only app. WE don't need to support them here.

**Comment by tahini** [comment]
I was hoping to postpone fixing #442 to later when we have the new infra up... but ok, I can do it now.

**Comment by tahini** [comment]
@greenscientist So these are brand new migrations with the table creation. chaire-lib-backend and transition-backend both have migration tables so they are all independent. chaire-lib's must be run before transition-backend.

Tested both on new project and migration from transition-legacy.

File: packages/chaire-lib-backend/src/scripts/config/setup.task.ts [hunk 2]
```diff
@@ -8,11 +8,10 @@ import 'chaire-lib-common/lib/config/shared/dotenv.config'; // Unused, but must
 import config from '../../config/server.config';
 import knex from 'knex';
 
-// These files should not be published, they are for application preparation and should never be called by actual app
```
Comment (greenscientist) [inline packages/chaire-lib-backend/src/scripts/config/setup.task.ts:11]: Reasking here, why is this comment not applicable anymore? Just curious

**Review by greenscientist — APPROVED** [review]
Sounds good

**Pull request merged**